packages/@react-types/provider/src/index.d.ts

```diff
@@ -20,22 +20,57 @@ export interface Theme {
 }
 
 interface ContextProps {
+  /**
+   * Set the placement of the toast alerts for the provider.
+   * default "bottom"
+   */
   toastPlacement?: ToastPlacement,
+  /** Sets quiet property for children components that use this property via context. Sets the quiet style. */
   isQuiet?: boolean,
+  /** Sets emphasized property for children component that use this property via context. Sets the quiet style. */
   isEmphasized?: boolean,
+  /** Sets disabled property for children component that use this property via context. Disables the component. */
   isDisabled?: boolean,
+  /** Sets required property for children component that use this property via context. Sets the required style. */
   isRequired?: boolean,
+  /** Sets read only property for children component that use this property via context. Component is read only. */
   isReadOnly?: boolean,
+  /** Sets validation state property for children component that use this property via context. Sets the validation state style. */
   validationState?: ValidationState
 }
 
 export interface ProviderProps extends ContextProps, DOMProps, StyleProps {
+  /** The components to receive provider properties and context. */
   children: ReactNode,
+  /**
+   * Spectrum theme scoped to this provider and it's children components.
+   * @default "light"
+   */
   theme?: Theme,
-  colorScheme?: ColorScheme, // by default, chooses based on OS setting
-  defaultColorScheme?: ColorScheme, // if no OS setting, which to choose
-  scale?: Scale, // by default, chooses based on touch/mouse
+  /**
+   * Color scheme scoped to this provider and it's children components.
+   * Defaults to the color scheme set by the OS.
+   */
+  colorScheme?: ColorScheme,
+  /** If there is not an OS color scheme this is the default. */
+  defaultColorScheme?: ColorScheme,
+  /**
+   * Spectrum scale scoped to this provider and it's children components.
+   * By default this is selected based on touch or mouse pointer type of the OS.
+   * @default "medium"
```

Comment:
I dont think you should have the this default here because there is not really a single default. The default changes. So probably better to not have this documented as one